apps/web/src/pages/ApplicantDashboardPage/ApplicantDashboardPage.tsx

```diff
@@ -33,6 +40,91 @@ export const ApplicantDashboardPage_Fragment = graphql(/* GraphQL */ `
     poolCandidates {
       ...ApplicationsProcessesTaskCard
     }
+    lookingForEnglish
+    lookingForFrench
+    lookingForBilingual
+    estimatedLanguageAbility {
+      value
+      label {
+        en
+        fr
+      }
```

Comment:
I think it is safe to remove all the label parts here and just do `enumName { value }`

Reply:
Okay, looks like we need these to keep typescript happy for now. Created a ticket to address the underlying issue.

https://github.com/GCTC-NTGC/gc-digital-talent/issues/12736